benchexec/tools/swat.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+from benchexec.tools.template import BaseTool2
+
+
+class Tool(BaseTool2):
+    """
+    Tool info module for SWAT a tool ensemble for the dynamic symbolic execution of modern Java.
+    """
```

Comment:
Is there also a tool webpage that could be linked here? "SWAT" is unfortunately a very generic name that seems impossible to look up via a web search. Even a search for "swat tool ensemble for the dynamic symbolic execution of modern Java" does not seem to find your tool, only GDart?!

Reply:
Hi, it was a rather late decision to participate, so we have not set up a website or a proper repository. 
Our plan is to do that by the final deadline on the 22nd. But we don't have one for the pre-run. 

If it's mandatory, I can set up a quick website on our institute's homepage, but for now it would be rather empty.

Reply:
If you already know what the URL of the tool webpage will be in the future, you can put it here already. Otherwise please file another PR later to update it.

Reply:
I've added a link to the institutes website as a first draft :)

We will do an additional PR until the 22nd!